src/config/src/utils.ts

```diff
@@ -86,3 +93,59 @@ export async function fetchRulesLabels(config: Pick<SigynConfig, "loki" | "rules
 
   return labels;
 }
+
+export function applyDefaultValues(
+  config: PartialSigynConfig | SigynConfig
+): SigynConfig {
+  return {
+    ...config,
+    missingLabelStrategy: config.missingLabelStrategy ?? kDefaultMissingLabelStrategy,
+    defaultSeverity: config.defaultSeverity ?? kDefaultAlertSeverity,
+    rules: config.rules.map((rule) => {
+      if (rule.polling === undefined) {
+        rule.polling = kDefaultRulePolling;
+      }
```

Comment:
```suggestion
      rule.polling ??= kDefaultRulePolling
```
Pareil pour le reste